utils/generate_standalone.py

```diff
@@ -87,6 +87,8 @@ def main():
                         help='chemiscope JSON input that is appended to provide default data')
     parser.add_argument('-o', '--output', type=str, default="",
                         help='output file name. Defaults to chemiscope.html or the name of the JSON')
+    parser.add_argument('-l', '--loader', default=False, action='store_true',
```

Comment:
I don't think this flag warrant a short form, I usually prefer command line arguments to be more explicit.

Reply:
fine with me